lib/internal/modules/esm/get_format.js

```diff
@@ -161,14 +161,14 @@ function getFileProtocolModuleFormat(url, context = { __proto__: null }, ignoreE
       default: { // The user did not pass `--experimental-default-type`.
         // `source` is undefined when this is called from `defaultResolve`;
         // but this gets called again from `defaultLoad`/`defaultLoadSync`.
-        let parsedSource;
-        if (source) {
-          const { stripTypeScriptTypes } = require('internal/modules/helpers');
-          parsedSource = stripTypeScriptTypes(source, url);
-        }
+        // Since experimental-strip-types depends on detect-module, we always return null
+        // if source is undefined.
+        if (!source) { return null; }
+        const { stripTypeScriptTypes, stringify } = require('internal/modules/helpers');
+        const stringifiedSource = stringify(source);
+        const parsedSource = stripTypeScriptTypes(stringifiedSource, fileURLToPath(url));
```

Comment:
That `fileURLToPath` will throw if `url` is not using the `file:` scheme

Reply:
it's in `getFileProtocolModuleFormat` so I assume its always gonna be a file 😟